app/controllers/application_controller.rb

```diff
@@ -37,6 +37,11 @@ def pundit_scope
                  status: :not_found
   end
 
+  rescue_from ActiveRecord::RecordInvalid do |error|
+    logger.info "#{error.message} (#{error.class})"
+    render_error(error.record || error.message)
```

Comment:
What is the case when `record` is `nil`?

Reply:
I don't know when it happens, however the `record` can be nil. Here is the implemenation: https://github.com/rails/rails/blob/v5.2.4.3/activerecord/lib/active_record/validations.rb#L24

Reply:
Note, that it had to be changed to use `render_model_errors` if the record is set.